api/README.md

```diff
@@ -0,0 +1,73 @@
+<p align="center">
```

Comment:
remove this as this is the default one which is our project is now a bit diverged